client/src/parsec/claim_device.ts

```diff
@@ -0,0 +1,243 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import {
+  libparsec,
+} from '@/plugins/libparsec';
+
+import {
+  DeviceLabel,
+  DeviceClaimFinalizeInfo,
+  DeviceClaimInProgress1Info,
+  DeviceClaimInProgress2Info,
+  DeviceClaimInProgress3Info,
+  AvailableDevice,
+  ClientEvent,
+  Handle,
+  Result,
+  DEFAULT_HANDLE,
```

Comment:
DEFAULT_HANDLE could be moved to `internals` too, but we can leave it to another PR.